com.wps.Office.yml

```diff
@@ -68,8 +68,8 @@ modules:
         only-arches:
           - x86_64
         url: https://wdl1.pcfg.cache.wpscdn.com/wpsdl/wpsoffice/download/linux/10161/wps-office_11.1.0.10161.XA_amd64.deb
-        sha256: 8ca6084428693c81833f6fdf13442a9ea9052638d5bdc0e4d7512075f40007c3
-        size: 228984666
+        sha256: c07038a25deba101b5e9da735cbf8526d9ae1a2e497607cbcd1116d63da0d046
+        size: 93444770
```

Comment:
@wjt So it looks like incomplete read indeed. Why it's not cached? Tests do properly catch incomplete read from httpbingo. I'm out of ideas.